var/spack/repos/builtin/packages/perl-perl-unsafe-signals/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PerlPerlUnsafeSignals(PerlPackage):
```

Comment:
I wouldn't mind if this were called `PerlUnsafeSignals`

Reply:
I know, but the package is called 'Perl-Unsafe-Signals`. A similar thing occurs with a few R packages and we made those consistent with the upstream name. I am fine either way though.

Reply:
Oh, were you referring to the package name or just the class name?

Reply:
The class name determines the package name (as mentioned in https://github.com/spack/spack/pull/12904#discussion_r328849743).

Reply:
And to be clear I'm OK with the name as is. I just didn't want you to feel that this name was mandatory. It does help find duplicate packages (but as mentioned elsewhere there are minor inconsistencies so I search based on URL components instead typically).